### PR TITLE
added a course, a cheatsheet, a tutorial (changes made)

### PR DESCRIPTION
**Pull request by MugdhaBehere**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description

Added resources under the following categories:
Interactive Tutorials: SQLZoo
Free English Courses: Stanford CS224N, Stanford CS230
Free Programming Cheat Sheet: The Big-O Cheat Sheet.
(I had previously raised a pull request(it was numbered #9783) for this purpose and changes were requested, I have made the changes, created a new branch, and raised a new pull request as required by you.)

### Why is this valuable (or not)?

SQLZoo is interactive and free.
CS224N and CS230 are courses by Stanford University which makes it valuable.
The Big-O Cheat Sheet is comprehensive and covers the complexities of many algorithms.

### How do we know it's really free?

We can browse the complete website of SQLZoo for free without encountering any webpage that asks us to pay.
Stanford CS224N and Stanford CS230 are courses with their content released and their content available for free on Stanford's website and YouTube Channel.
The website for Big-O Cheat Sheet can also be completely browsed for free without encountering any webpage that asks us to pay.

### For book lists, is it a book? For course lists, is it a course? etc.

SQLZoo is an interactive tutorial.
Stanford CS224N and CS230 are courses.
The Big-O Cheat Sheet is a Cheat Sheet.

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -535,6 +535,8 @@
 * [Practical Deep Learning For Coders taught](http://www.fast.ai) - Jeremy Howard
 * [Practical Deep Learning for Coders, v3 (using fastai library)](https://course.fast.ai)
 * [Self-Paced Courses for Deep Learning](https://developer.nvidia.com/deep-learning-courses)
+* [Stanford CS 224N: Natural Language Processing with Deep Learning](https://www.youtube.com/playlist?list=PLoROMvodv4rOSH4v6133s9LFPRHjEmbmJ) (Stanford University)
```
Comment (eshellman) [inline courses/free-courses-en.md:538]:
```suggestion
* [Stanford CS 224N: Natural Language Processing with Deep Learning](https://www.youtube.com/playlist?list=PLoROMvodv4rOSH4v6133s9LFPRHjEmbmJ) (Stanford Online)
```

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -535,6 +535,8 @@
...
+* [Stanford CS230: Deep Learning](https://www.youtube.com/playlist?list=PLoROMvodv4rOABXSygHTsbvUz4G_YQhOb) (Stanford University)
```
Comment (eshellman) [inline courses/free-courses-en.md:539]:
```suggestion
* [Stanford CS230: Deep Learning](https://www.youtube.com/playlist?list=PLoROMvodv4rOABXSygHTsbvUz4G_YQhOb) (Stanford Online)
```

**Review by eshellman — CHANGES_REQUESTED** [review]
Thanks for adding this PR

File: courses/free-courses-en.md (hunk 2 above)
Comment (MugdhaBehere) [reply courses/free-courses-en.md]: These changes have been made

File: courses/free-courses-en.md (hunk 1 above)
Comment (MugdhaBehere) [reply courses/free-courses-en.md]: These changes have been made

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by MugdhaBehere** [comment]
@eshellman Sir, could you please label this pull request as hacktoberfest-accepted?  Also, please don't close the pull request because these are requirements for the pull request to be considered for Hacktoberfest 2023, as the website mentions. (https://hacktoberfest.com/participation/#pr-mr-details)

**Pull request merged**